batch/kueue-cohort/create_jobs.sh

```diff
@@ -15,6 +15,10 @@
 #!/bin/bash
 while :
 do
+  kubectl create -f ${1}
+  kubectl create -f ${1}
+  kubectl create -f ${1}
+  kubectl create -f ${1}
```

Comment:
did you intend to have all these lines? You should be able to control the rate by changing the sleep time